File: 0387.字符串中的第一个唯一字符/0387-字符串中的第一个唯一字符.cpp
class Solution {
public:
    int firstUniqChar(string s) {
        string dic = "abcdefghijklmnopqrstuvwxyz";
        int res = s.size();
        for (int i = 0; i < dic.size(); ++i){
            int a = s.find(dic[i]);
            int b = s.rfind(dic[i]);
            cout << a <<" " << b <<endl;
            if (a == b && a != -1){
                res = a<res ? a :res;
            }
        }
        return res < s.size() ? res : -1;
    }
};

File: 0387.字符串中的第一个唯一字符/0387-字符串中的第一个唯一字符.py
class Solution(object):
    def firstUniqChar(self, s):
        """
        :type s: str
        :rtype: int
        """
        dic = collections.Counter(s)
        
        for i, ch in enumerate(s):
            if dic[ch] == 1:
                return i
        return -1